Server/mods/deathmatch/logic/CColPolygon.cpp

```diff
@@ -34,6 +34,11 @@ bool CColPolygon::DoHitDetection(const CVector& vecNowPosition)
     if (!IsInBounds(vecNowPosition))
         return false;
 
+    float z = vecNowPosition.fZ;
+
+    if (z > m_fCeil || z < m_fFloor)
```

Comment:
Unnecessary variable.
```suggestion
    if (vecNowPosition.fZ > m_fCeil || vecNowPosition.fZ < m_fFloor)
```